RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Credential Issuer common libraries Release Notes
 
+## 3.2.0
+    - Updated audit event tests to use the new test harness events endpoint
+    - Added AWS credentials to sign the HTTP request for the events endpoint
+    - Removed the previous SQS and cloudformation usages for TxMA step definitions
```

Comment:
Could you clarify if the last point is relevant, reading this I'd something to have been removed from this repo?